account_commission/models/account_move.py

```diff
@@ -21,6 +21,33 @@ class AccountMove(models.Model):
         compute="_compute_agents",
         search="_search_agents",
     )
+    settlement_count = fields.Integer(compute="_compute_settlement")
+    settlement_ids = fields.Many2many(
```

Comment:
Should this field technically be one2many given that there should be only one valid vendor bill for a settlement?

We may take this chance to update _compute_invoice_id() of commission.settlement to exclude cancelled vendor bills as well.
https://github.com/oca/commission/blob/b6715536aa7c115b9283d07e10f7ffae51268ee2/account_commission/models/commission_settlement.py#L44-L47

@pedrobaeza Any thoughts on this?

Reply:
OK for me.